res/controllers/Ion-Discover-DJ-Pro.midi.xml

```diff
@@ -0,0 +1,797 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="2.1.0+" schemaVersion="1">
+    <info>
+        <name>Ion Discover DJ Pro</name>
+        <author>joachim,meltedpianoman</author>
+        <description>This preset provides functionality for the Ion Discover DJ Ppro controller for Mixxx</description>

```

Comment:
Not really descriptive, maybe something like this? 

```suggestion
        <description>A mapping for the Ion Discover DJ Pro, a 2-deck controller with integrated audio interface and iPhone dock.</description>

```

Reply:
Much better indeed